src/Umbraco.Core/Models/Identity/IdentityMapDefinition.cs

```diff
@@ -11,14 +12,24 @@ public class IdentityMapDefinition : IMapDefinition
         private readonly ILocalizedTextService _textService;
         private readonly IEntityService _entityService;
         private readonly IGlobalSettings _globalSettings;
+        private readonly AppCaches _appCaches;
 
+        [Obsolete("Use constructor specifying all dependencies")]
         public IdentityMapDefinition(ILocalizedTextService textService, IEntityService entityService, IGlobalSettings globalSettings)
```

Comment:
I guess this ctor should call the new non-obsolete ctor and pass in Current.AppCaches? Otherwise it seems like a NullReferenceException